rpm/caddy.spec

```diff
@@ -0,0 +1,134 @@
+# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
+%global _dwz_low_mem_die_limit 0
+
+Name:           caddy
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_versioning_prereleases_with_tilde
+Version:        2.0.0~beta6
+%global tag     v2.0.0-beta6
+Release:        1%{?dist}
+Summary:        HTTP/2 web server with automatic HTTPS
+License:        ASL 2.0
+URL:            https://caddyserver.com
+
+# In order to build Caddy with version information, we need to import it as a
+# go module.  To do that, we are going to forgo the traditional source tarball
+# and instead use this simple go file.  This method requires that we allow
+# networking in the build environment.
+Source0:        main.go
+# Use official resources for config, unit file, and welcome page.
+# https://github.com/caddyserver/dist
+Source1:        https://raw.githubusercontent.com/caddyserver/dist/master/config/Caddyfile
+Source2:        https://raw.githubusercontent.com/caddyserver/dist/master/init/caddy.service
+Source3:        https://raw.githubusercontent.com/caddyserver/dist/master/welcome/index.html
+# Since we are not using a traditional source tarball, we need to explicitly
+# pull in the license file.
+Source4:        https://raw.githubusercontent.com/caddyserver/caddy/%{tag}/LICENSE
+
+BuildRequires:  golang >= 1.13
+BuildRequires:  git-core
+BuildRequires:  systemd
+%{?systemd_requires}
+Provides:       webserver
+
+
+%description
+Caddy is the HTTP/2 web server with automatic HTTPS.
```

Comment:
(Here too, a more generic description will be forthcoming. We should chat about what a standard project description should be across all packages.)